### DISCUSSION TITLE
Not an issue: kudos

### DISCUSSION THREAD

**Discussion by gklimowicz** [alphapapa/burly.el#67] [General]
Just so you are aware: There are people using this package, and loving it.

Thank you for this. I find it works (generally) great. That it meshes so well with bookmarks is very cool.

When I can determine whether my minor problems are due burly, I'll submit an issue or two.

(I suspect my problems are with me...)

**Comment by alphapapa** [comment]
Hi Gary,

Thanks very much for the kind words.  I'm glad to hear that it's useful.  Please do let me know if you have specific feedback.
